pysb/importers/bngl.py

```diff
@@ -65,7 +66,8 @@ def _eval_in_model_env(self, expression):
 
         # Quick security check on the expression
         if re.match(r'^[\w\s()/+\-._*]*$', expression):
-            return parse_expr(expression, self._model_env)
+            return sympy.sympify(expression, locals=self._model_env,
```

Comment:
Why the change here?

Reply:
This is now altered to use `parse_bngl_expr`